docs/api/orders.rst

```diff
@@ -3,165 +3,89 @@ Orders API
 
 Sylius orders API endpoint is `/api/v1/orders`.
 
-Index of all orders
--------------------
+If you request an order, you will receive an object with following fields:
+
+
++-----------------------+---------------------------------------------+
+| Field                 | Description                                 |
++=======================+=============================================+
+| id                    | Id of cart                                  |
++-----------------------+---------------------------------------------+
+| items                 | List of items related to cart               |
++-----------------------+---------------------------------------------+
+| items_total           | Sum of all items prices                     |
++-----------------------+---------------------------------------------+
+| adjustments           | List of adjustments related to cart         |
++-----------------------+---------------------------------------------+
+| adjustments_total     | Sum of all order adjustments                |
++-----------------------+---------------------------------------------+
+| total                 | Sum of items total and adjustments total    |
++-----------------------+---------------------------------------------+
+| customer              | `Customer detailed serialization`_ for cart |
++-----------------------+---------------------------------------------+
+| channel               | `Default channel serialization`_            |
++-----------------------+---------------------------------------------+
+| currency_code         | Currency of a cart                          |
++-----------------------+---------------------------------------------+
+| checkout_state        | `State of checkout process`_                |
++-----------------------+---------------------------------------------+
+| state                 | `One of available order states`_            |
++-----------------------+---------------------------------------------+
+| checkout_completed_at | Date when the checkout has been completed   |
++-----------------------+---------------------------------------------+
+| number                | Number of an order                          |
++-----------------------+---------------------------------------------+
+| shipping_address      | Detailed address serialization              |
++-----------------------+---------------------------------------------+
+| billing_address       | Detailed address serialization              |
```

Comment:
This one and the above one have identical descriptions, but shouldn't.

`Detailed billing address serialization`